omnibus/config/software/erlang.rb

```diff
@@ -37,6 +37,7 @@
 relative_path "otp-OTP-#{version}"
 
 # versions_list: https://github.com/erlang/otp/tags filter=*.tar.gz
+version('24.3.4')    { source sha256: '76fcca5ba6f11eb9caac32bf053badc46b5d66f867150eef077f4f0d7944ecd7' }
```

Comment:
this seems to be the sha for otp_src_24.3.4.tar.gz

the .tar.gz file is `e59bedbb871af52244ca5284fd0a572d52128abd4decf4347fe2aef047b65c58`

Reply:
```suggestion
version('24.3.4')    { source sha256: 'e59bedbb871af52244ca5284fd0a572d52128abd4decf4347fe2aef047b65c58' } 
```